src/components/progressCircular/js/progressCircularDirective.js

```diff
@@ -47,13 +47,22 @@ angular
     '$$rAF',
     '$window',
     '$mdProgressCircular',
+    '$mdTheming',
```

Comment:
Good question. I suppose I just went with it out of habit, I'll switch to the regular DI.